examples/quickstart-diffusion/README.md

```diff
@@ -0,0 +1,201 @@
+---
+tags: [quickstart, diffusion, vision, federated-learning]
+dataset: [Oxford-Flowers]
+framework: [diffusers, peft, flower]
+---
+
+
+# Federated Diffusion Model Training with Flower (Quickstart Example)
+
+This example demonstrates how to train a **Diffusion Model** (based on [Stable Diffusion v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5)) in a **Federated Learning (FL)** environment using [Flower](https://flower.ai/).  
+The training uses **Low-Rank Adaptation (LoRA)** to enable lightweight fine-tuning of a diffusion model in a distributed setup, even with limited compute resources.
+The example uses the Oxford Flowers dataset, a collection of RGB flower images commonly used for training image-generation models.
+
+Prompt:
+A realistic image of a blooming yellow daffodil in natural sunlight
+
+The model was trained using 400 images for training, 80 images for evaluation, and fine-tuned for 5 federated rounds.
+Below is a sample output image generated using the final LoRA-adapted model.
+
+![](_static/federated_diffusion_sample.png)
+---
+## Overview
+
+In this example:
+
+- Each client fine-tunes **only the LoRA parameters** of the Stable Diffusion UNet.
+- The **Oxford-Flowers** dataset is partitioned among multiple clients using Flower Datasets.
+- The server performs **FedAvg aggregation** on the LoRA weights after each round.
+- After all rounds, the aggregated LoRA adapter is saved into `final_lora_model/` and can be merged with the base model for image generation.
+
+This provides a clean example of how **federated diffusion fine-tuning** can be performed using Diffusers + PEFT + Flower.
+
+---
+## Set up the project
+
+### Clone the project
+
+Start by cloning the example project. We prepared a single-line command that you can copy into your shell which will checkout the example for you:
+
+```shell
+git clone --depth=1 https://github.com/adap/flower.git _tmp \
+		&& mv _tmp/examples/quickstart-diffusion . \
+		&& rm -rf _tmp && cd quickstart-diffusion
+```
+
+After cloning, your directory will look like this:
+
+```shell
+quickstart-diffusion
+├── diffusion_example
+│   ├── __init__.py
+│   ├── client_app.py        # Defines your ClientApp logic
+│   ├── server_app.py        # Defines the ServerApp and strategy
+│   └── task.py              # Model setup, data loading, and training functions
+├── pyproject.toml           # Project metadata and dependencies
+└── README.md                # This file
+
+
+```
+### Install dependencies and project
+
+First, create and activate a virtual environment (recommended to avoid dependency conflicts):
+
+```bash
+# Create environment with Python 3.9 (you may also use 3.10 or 3.11)
+conda create -n diffusion-env python=3.9 -y
+```
+```bash
+conda activate diffusion-env
+```
+
+Install the dependencies defined in `pyproject.toml` as well as the `diffusion_example` package.
```

Comment:
```suggestion
Install the dependencies defined in `pyproject.toml` as well as the `diffusion_example` package:
```

In the examples we prefer not mention a specific way of creating an environment. We can keep just this last line